src/app/banner/banner.component.ts

```diff
@@ -9,10 +13,32 @@ import { environment } from '../../environments/environment';
 })
 export class BannerComponent implements OnInit {
   showBanner: boolean;
-  constructor() { }
+  mismatchedVersion = false;
+  versionFromAPI: string;
+  versionBuiltWith: string;
+
+  constructor(private gA4GHService: GA4GHService) { }
 
   ngOnInit() {
     this.showBanner = environment.staging;
+
+    this.gA4GHService.metadataGet()
```

Comment:
this looks like it does a 2nd metadataGet call when the footer already did one.  double check if there's 1 or 2 metadata get calls every time the application is loaded.  If it's duplicated, somehow make the footer and banner use metadata from the same call.